ui/src/components/Form/DynamicForm.vue

```diff
@@ -10,15 +10,13 @@
         :hint="field.short"
         :value="field.value"
       />
-
       <Boolean
         v-if="field.type == 'boolean'"
         :label="field.label"
         ref="active"
         :hint="field.short"
-        :value="field.value"
+        :value="isActivePractitioner==true?true:this.value"
```

Comment:
This component will be used for more than just the add person page. Can you update the default value there and pass it in here? Then you won't need this conditional.

Also, it should be field.value, not this.value.